tests/services/detectionlists/test_departing_employee.py

```diff
@@ -65,29 +66,27 @@ def test_choices_are_correct(self):
 
 class TestDepartingEmployeeClient(object):
     @pytest.fixture
-    def mock_get_all_response(self, py42_response):
-        py42_response.text = _GET_ALL_RESPONSE
-        return py42_response
+    def mock_get_all_response(self, mocker):
+        return py42_response(mocker, _GET_ALL_RESPONSE)
```

Comment:
This is not really on you, but this `return` should probably be a `yield`, if we are being picky about types, which _can_ matter sometimes.